README.md

```diff
@@ -75,6 +75,12 @@ Bergamot translator provides a unified API for ([Marian NMT](https://marian-nmt.
             emmake make -j
             ```
 
+    3. Enable SIMD Wormhole via Wasm instantiation API in generated artifacts
```

Comment:
With the introduction of this step, please update the phrase further down in this readme that says "As long as you don't update any submodule, just follow steps in 4.ii to recompile."

Reply:
Done.